### ISSUE TITLE
Multiple cursors on different buffers

### ISSUE THREAD

**Issue by YoungFrog** [magnars/multiple-cursors.el#243]
I know I should already be happy to have multiple cursors, but I'd like to have even more : is it possible to act on fake cursors in different buffers/windows at the same time?

**Comment by magnars** [comment]
Yes, that should be theoretically possible. I'm sure there are quite a few snags tho. It's not something I'm interested in pursuing. Guess I'd use macros instead for such a use case. :)
